xformers/ops/fmha/_triton/splitk_kernels.py

```diff
@@ -454,14 +447,14 @@ def _fwd_kernel_splitK(
                 V_scale_shift_block_ptr = None
             logical_block_idx += 1
 
-        k: "VAR_ARGS_ARRAY"  # noqa: F821
-        v: "VAR_ARGS_ARRAY"  # noqa: F821
+        qk = tl.zeros([BLOCK_M, BLOCK_N], dtype=tl.float32)
         for i in range(len(acc)):  # noqa: F821
-            k[i], v[i] = load_dequantize_k_v_group(  # noqa: F821
+            q_i = tl.load(  # noqa: F821
```

Comment:
What is the advantage of loading q in each iteration instead of load once at the beginning?